### PR TITLE
Take obsprep config info out of dict and into config yaml

### LINKED ISSUE #781: transfer obsproc info in ush/soca/prep_marine_obs.py to  parm/soca/obsproc/obsproc_config.yaml
The script `ush/soca/prep_marine_obs.py` has a dictionary with laboriously collected obsproc info about incoming observation files that belongs in `parm/soca/obsproc/obsproc_config.yaml` - the dictionary containing this info is obsolete, but the info is not

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
Left over observer info for obs prep was still in a dict in ush/soca/prep_marine_obs.py, now in  
parm/soca/obsprep/obsprep_config.yaml with supplementary info

Resolves https://github.com/NOAA-EMC/GDASApp/issues/781

### CODE REVIEW

**Review by apchoiCMD — DISMISSED** [review]

**Review by ShastriPaturi — DISMISSED** [review]

File: parm/soca/obsprep/obsprep_config.yaml [hunk 1]
```diff
@@ -39,3 +39,53 @@ observations:
       units: C
       min: -3.0
       max: 50.0
+- obs space:
```
Comment (guillaumevernieres) [inline parm/soca/obsprep/obsprep_config.yaml]: What's below is not tested, so remove for now. What's above does not really work yet. I'd suggest not touching this file for now.

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
See comment

File: parm/soca/obsprep/obsprep_config.yaml (hunk 1 above)
Comment (AndrewEichmann-NOAA) [reply parm/soca/obsprep/obsprep_config.yaml]: I can take it all out, but what do you mean "does not really work yet"?

Comment (guillaumevernieres) [reply parm/soca/obsprep/obsprep_config.yaml]: Related to what we talked about earlier, and somewhat "documented" here: https://github.com/NOAA-EMC/GDASApp/issues/869

Comment (AndrewEichmann-NOAA) [reply parm/soca/obsprep/obsprep_config.yaml]: I removed the untested entries. Fixing the regexes will require more complicated testing and should be done later. Also, obsolete "output file" entries were removed (these are created automatically in the script)

**Review by guillaumevernieres — DISMISSED** [review]
I forgot about this PR @AndrewEichmann-NOAA , sorry! Looks good.

**Comment by guillaumevernieres** [comment]
Could you fix the code norm @AndrewEichmann-NOAA ?

**Comment by AndrewEichmann-NOAA** [comment]
@guillaumevernieres done

**Review by guillaumevernieres — APPROVED** [review]

**Pull request merged**